ledger/catchupaccessor.go

```diff
@@ -179,16 +181,17 @@ type CatchupAccessorClientLedger interface {
 // MakeCatchpointCatchupAccessor creates a CatchpointCatchupAccessor given a ledger
 func MakeCatchpointCatchupAccessor(ledger *Ledger, log logging.Logger) CatchpointCatchupAccessor {
 	return &catchpointCatchupAccessorImpl{
-		ledger:        ledger,
-		stagingWriter: &stagingWriterImpl{wdb: ledger.trackerDB().Wdb},
-		log:           log,
+		ledger:          ledger,
+		catchpointStore: store.NewCatchpointSQLReaderWriter(ledger.trackerDB().Wdb.Handle),
```

Comment:
in theory we might use separate reader and writer handles. for sqlite it does not matter but still